src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.PathologyRecordFixtures.generatePathologyRecord;
+
+@ExtendWith(MockitoExtension.class)
+public class BundleMapperTest {
+
+    @Test
+    public void testMapMessageToPathologyRecord() {
+        final BundleMapper bundleMapper = new BundleMapper();
+
+        final Bundle bundle = bundleMapper.mapToBundle(generatePathologyRecord());
+
+        assertNotNull(bundle.getMeta().getLastUpdated());
+        assertEquals(bundle.getMeta().getProfile().get(0).asStringValue(), "https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1");
```

Comment:
I believe your asserts should be the other way around, `assertEquals(expected, actual)`
```suggestion
        assertEquals("https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1", bundle.getMeta().getProfile().get(0).asStringValue());
```